### PR TITLE
show events in map & show current position user

### PR DESCRIPTION
**Pull request by nicolascr181**
image

### CODE REVIEW

File: dashboard/src/components/Directory/Map.vue [hunk 1]
```diff
@@ -1,52 +1,78 @@
...
+    LIcon,
     ShowEvent
   },
   mounted() {
     // Get events
     getJSON("/events", null, false)
       .then((res) => {
-        console.log(res);
+        this.events = res.data.items;
+        console.log(this.events)
```
Comment (1701722767) [inline dashboard/src/components/Directory/Map.vue]: delete this

File: dashboard/src/components/Directory/Map.vue [hunk 2]
```diff
@@ -1,52 +1,78 @@
...
     LMarker,
-    LPopup,
+    LTooltip,
+    LIcon,
     ShowEvent
   },
   mounted() {
     // Get events
     getJSON("/events", null, false)
       .then((res) => {
```
Comment (1701722767) [inline dashboard/src/components/Directory/Map.vue:68]: validate that not return error

File: dashboard/src/components/Directory/Map.vue [hunk 3]
```diff
@@ -1,52 +1,78 @@
...
-        console.log(res);
+        this.events = res.data.items;
+        console.log(this.events)
       })
       .catch((err) => {
-        console.log(err);
+
```
Comment (1701722767) [inline dashboard/src/components/Directory/Map.vue]: show error when anything fail

File: dashboard/src/components/Directory/Map.vue [hunk 4]
```diff
@@ -70,6 +105,17 @@ export default {
         const param = { composite_key : keys };
         this.$refs.ShowEvent.show(param);
       },
+
+      onReady() {
+        this.map = this.$refs["map"].mapObject;
+        this.map.locate();
+      },
+
+      onLocationFound(l) {
```
Comment (1701722767) [inline dashboard/src/components/Directory/Map.vue]: what is l?

File: dashboard/src/components/Directory/Map.vue [hunk 5]
```diff
@@ -70,6 +105,17 @@ export default {
...
+        console.log(l);
```
Comment (1701722767) [inline dashboard/src/components/Directory/Map.vue]: delete this

**Review by sergiomonz — APPROVED** [review]

File: dashboard/src/components/Events/Register.vue [hunk 6]
```diff
@@ -285,6 +266,17 @@ export default {
           });
         });
     },
+
+    transformDate(date){
```
Comment (1701722767) [inline dashboard/src/components/Events/Register.vue]: this can be a helper?

**Review by 1701722767 — CHANGES_REQUESTED** [review]

**Review by 1701722767 — APPROVED** [review]

**Pull request merged**